crates/orchestrator/src/config.rs

```diff
@@ -713,7 +713,7 @@ impl<KEY: SignatureKey> From<HotShotConfigFile<KEY>> for HotShotConfig<KEY> {
     }
 }
 /// default number of rounds to run
-pub const ORCHESTRATOR_DEFAULT_NUM_ROUNDS: usize = 10;
+pub const ORCHESTRATOR_DEFAULT_NUM_ROUNDS: usize = 100;
 /// default number of transactions per round
```

Comment:
The changes look good, but can we also remove `da_web_server_config` and `web_server_config` from this file?

Reply:
I also removed them in `run-config.toml`